### PR TITLE
Clean up localizations

### PR DESCRIPTION
**Pull request by daniel-beck**
### Proposed changelog entries

* Remove Korean entries from Ukrainian localization.

Could probably skip; most other interesting changes are (also, mostly) implemented in #2813. And the bulk of this is removing "English to English" localizations, which doesn't seem notable.

---

So :rage: right now that this obvious nonsense was even committed. Even a cursory review reveals major problems.

* Remove "localized" strings that are just English
* Remove obvious vandalism
* Remove obviously wrong translations (Korean script in Ukrainian localization? wtf?)
* Remove Punjabi (pa_IN), and Shqip (sq, Albanian) localizations entirely.
  * I can't get anything useful out of Google Translate for the former. Not a strong reason to delete, but it's only a handful of translations (23) anyway, and I doubt THEY WRITE EVERYTHING IN ALL CAPS.
  * The latter is just [obvious vandalism](https://github.com/jenkinsci/jenkins/blob/e8efc96f3d4ce3ed6dc121661423ad4a2892e2b2/core/src/main/resources/hudson/model/AbstractBuild/index_sq.properties#L3).

I also fixed a few obvious typos.

### CODE REVIEW

**Comment by batmat** [comment]
heh, the first file I looked at was modified by me with https://github.com/daniel-beck/jenkins/commit/9ff7cfc4d8a3cedb32e6f73a9b47d04c653dae73#diff-1372ea548f450eecdfe94c4cae0f9c9a I hope it's not me :-), I guess not, and guess you refer to that email where every submitted translations since like 2 years was all committed in a batch :-/.

**Review by batmat — APPROVED** [review]
:+1: and congrats, having worked quite a bit on some translations, I know how cumbersome this kind of work can be.

File: core/src/main/resources/hudson/security/SecurityRealm/loginLink_fy_NL.properties [hunk 1]
```diff
@@ -1,3 +0,0 @@
...
...nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnbnbnbnbnbnbjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhjhyyyyyyyyyyyyyyyyyyyyyyyyyyyuuuuuuuuuuuuuuuuuuuuuuuuyuuuuuuuuuuuuuuuyuuuuuuuuuuuuuuuuuuuuuuuuuttttttttttttttttttttttttttcfdffffffffgggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
```
Comment (batmat) [inline core/src/main/resources/hudson/security/SecurityRealm/loginLink_fy_NL.properties]: wow, I wonder if really we have NL users? :-/

Comment (daniel-beck) [inline core/src/main/resources/hudson/security/SecurityRealm/loginLink_fy_NL.properties]: This isn't Dutch, but West Frisian, with 400k speakers.

**Comment by daniel-beck** [comment]
@batmat Well, the English "localizations" are mostly harmless. The rest is much worse.

**Comment by daniel-beck** [comment]
Also removed Gujarati here, as it seems to be written in two different scripts. No idea what's going on, but it's either getting removed here or in #2813.

**Review by oleg-nenashev — APPROVED** [review]
:+1: from what I see. I wish to say the day when @github starts rendering property files in the UI

**Pull request merged**